src/rexsdesktop/controller/User_.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package rexsdesktop.controller;
+
+import java.awt.image.BufferedImage;
+import rexsdesktop.CurrentUser;
+import rexsdesktop.model.UserModel;
+
+/**
+ *
+ * @author Eduardo
+ */
+public class User_{

```

Comment:
avoid this kind of name, maybe something like `UserController` it could be more readable

Reply:
I named it that way because there was already a class named User, sorry about that, the idea is to refactor all the User class in User_ and then change the name to User.